Tests/SpeziSchedulerTests/SchedulerTests.swift

```diff
@@ -286,18 +286,49 @@ final class SchedulerTests: XCTestCase {
         let taskId = UUID()
         let scheduledDate = Date()
         let scheduledEvent = Event(taskId: taskId, scheduledAt: scheduledDate)
-        
+
         let completedEvent = Event(taskId: taskId, scheduledAt: scheduledDate)
         completedEvent.complete(true)
-        
+
         var scheduledEventHasher = Hasher()
         scheduledEvent.hash(into: &scheduledEventHasher)
         let scheduledEventHash = scheduledEventHasher.finalize()
-        
+
         var completedEventHasher = Hasher()
         completedEvent.hash(into: &completedEventHasher)
         let completedEventHash = completedEventHasher.finalize()
-        
+
         XCTAssertEqual(scheduledEventHash, completedEventHash)
     }
+    func testTaskSchedulingWithStartDateAndEndDate() async throws {
```

Comment:
Please add empty lines between different functions:
```suggestion
    
    func testTaskSchedulingWithStartDateAndEndDate() async throws {
```